PerformanceRefactor.md

```diff
@@ -0,0 +1,43 @@
+# Performance Refactor Proposal
```

Comment:
Thank you to the authors for drafting this document. In terms of communicating (at a high level) the objectives and scope of the refactoring, it is good.

But  feel that the clarity would be enhanced by a sentence or two explaining _why_ the Performance Refactoring is being proposed. What is the goal of doing this refactoring. I mean, I think I know the main reasons (per my understanding, the reasons why the refactoring is being proposed are to reduce software maintenance and component-level testing costs going forward and to enable more rapid evolution of the Translator system, and to reduce ), but I feel that the document should summarize the reasons.